src/platforms/common/usb_serial.c

```diff
@@ -206,14 +213,12 @@ size_t debug_uart_write(const char *buf, const size_t len)
 
 	for (size_t i = 0; i < len && (usb_dbg_in + 1) % AUX_UART_BUFFER_SIZE != usb_dbg_out; ++i) {
 		if (buf[i] == '\n') {
-			usb_dbg_buf[usb_dbg_in++] = '\r';
-			usb_dbg_in %= AUX_UART_BUFFER_SIZE;
+			debug_serial_append_char('r');
```

Comment:
Should this not be `\r`? Or is it fixed in a later commit?

Reply:
It should and while it is fixed in a later commit, we've opted to edit this one and rebase so that the bug's not introduced in the first place.